src/lib.rs

```diff
@@ -3,16 +3,14 @@
 //!
 //! A stream can be any type that implements [`AsyncRead`], [`AsyncWrite`] and [`Unpin`].
 
-#![cfg_attr(docsrs, feature(doc_cfg))]
+#![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
```

Comment:
This is to prevent `#[cfg_attr(docsrs, doc(cfg(feature = "…")))]` everywhere in the code.